### ISSUE TITLE
False positive: return in finally block 

### ISSUE THREAD

**Issue by romani** [sevntu-checkstyle/sevntu.checkstyle#269]
Warning: "Forbid return statement in finally block: Finally block should not contain return statements."

https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/master/sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/ForbidReturnInFinalBlockCheck.java

```
	public void test() {
		try {
                         /// some code
		} finally { 
			// remove test data
			csf.doIn(new Callback<Void>() {
				@Override
				public Void execute(ISession session) {
					session.delete(this);
					return null;  // WARNING !!!!!!
				}
			}, "");
		}
	}
```
it is also required to rename Check, as block is "finally" not a "final"
